roaringbitmap/src/main/java/org/roaringbitmap/longlong/Roaring64NavigableMap.java

```diff
@@ -181,6 +205,29 @@ public Roaring64NavigableMap(
     resetPerfHelpers();
   }
 
+  /**
+   * set the serialization mode.
+   * if the serializationMode is successfully set, the static variable 'SERIALIZATION_MODE' will be ineffective.
```

Comment:
I do not understand this comment. I suppose you meant something like `setting the serializationMode on this bitmap, override the default behavior as defined per SERIALIZATION_MODE `

Reply:
ok